index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="#">
+                <img class="logo" src="/images/adesoji-dara-logo.png" alt="logo-image">
+            </a>
+            <img class="menu" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAAAXNSR0IArs4c6QAAAFFJREFUSEvtlEkKADAIA83/H52eu1APZQShnquTBaqAR/D9+IA04fqIbDuVdXkgaRK9OcABL+pPu/Ud4A7wDnAAHlF/AN4BDujfwXewJoD/pgNrVBgZX2h0sgAAAABJRU5ErkJggg==" alt="hambuger-icon" />
+        </nav>
+    </header>
+        <section class="header_content flex">
+            
+            <h1>
+                I'm Dara <br/> Glad to see you!
+            </h1>
+            <h3>I’m a software developer! </h3>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <ul class="social_icons flex">
```

Comment:
Good job with the unordered list but let's stick to the Figma template and add the missing logo and make the logo area look identical to the design.

<img width="282" alt="Screenshot 2023-01-16 at 11 24 19 PM" src="https://user-images.githubusercontent.com/25887253/212760653-d70823fe-48f1-4860-8c81-90b4e4dbda61.png">
<img width="282" alt="Screenshot 2023-01-16 at 11 24 32 PM" src="https://user-images.githubusercontent.com/25887253/212760655-8cc49904-8f3b-4473-86f6-d50734ee2110.png">